app/views/feeds/_form_audio_format.html.erb

```diff
@@ -16,28 +16,28 @@
 
         <div class="col-3 <%= display_bitrate(feed) %>" data-toggle-field-target="field" field="mp3 mp4">
           <div class="form-floating input-group">
-            <%= form.select :audio_bitrate, audio_bitrate_options %>
+            <%= form.select :audio_bitrate, options_for_select(audio_bitrate_options, 128) %>
             <%= form.label :audio_bitrate %>
           </div>
         </div>
 
         <div class="col-3 <%= display_bitdepth(feed) %>" data-toggle-field-target="field" field="wav flac">
           <div class="form-floating input-group">
-            <%= form.select :audio_bitdepth, audio_bitdepth_options %>
+            <%= form.select :audio_bitdepth, options_for_select(audio_bitdepth_options, 32) %>
```

Comment:
I'm not sure we want to set this - I think we only want default options for mp3, since that is the default format, and this is a field only for "wav flac"?

Reply:
This is the initially selected default when you select "flac" or "wav" from the leftmost dropdown.  So I do think we want it.

But I think 16 (or 24?) is more common than 32 for this.